docs/1.guide/5.options.md

```diff
@@ -56,6 +56,45 @@ Enabling this option allows multiple processes to bind to the same port, which i
 > [!NOTE]
 > Despite Node.js built-in behavior that has `exclusive` flag enabled by default, srvx uses non-exclusive mode for consistency.
 
+### `https`
+
+HTTPS configuration options. When provided, the server will use HTTPS instead of HTTP.
+
+**Example:**
+
+```js
+import { serve } from "srvx";
+import { readFileSync } from "node:fs";
+
+serve({
+  // Using file paths
+  https: {
+    key: readFileSync("./server.key"),
+    cert: readFileSync("./server.crt"),
```

Comment:
Since adapter are conditionally imported, we can support reading from a file path instead of making users to read it themselves.

Reply:
I see. Do you think we should add additional options like `keyPath`, `certPath`, etc., or should we assume that it will always be a file path?

Reply:
i would use simpler `key`/`cert` as path/url and then something like `inlineKey` for inline versions

Reply:
fixed! :)